src/client/fuse_ll.cc

```diff
@@ -1370,13 +1370,13 @@ const static struct fuse_lowlevel_ops fuse_ll_oper = {
 
 CephFuse::Handle::Handle(Client *c, int fd) :
   fd_on_success(fd),
-  client(c),
-  se(NULL),
+  client(c)
+{
 #if FUSE_VERSION < FUSE_MAKE_VERSION(3, 0)
-  ch(NULL),
-  mountpoint(NULL)
+  ch = nullptr;
```

Comment:
why not just initialize `ch` and `mountpoint` inlined, i.e. to initialize them at line 178 and line 179.

Reply:
Sounds good. Will fix it.